src/pages/Donations/Table.tsx

```diff
@@ -77,7 +77,11 @@ export default function Table({
           >
             USD Value
           </HeaderButton>
-          <>TX Hash</>
+          {status === "intent" ? (
+            <span className="flex justify-center">Finish Paying</span>
```

Comment:
Nit: We can drop the column header, since all the links have "Finish paying ->". Alternative is to drop the words from the links, but I think spelling out the links is better for accessibility and overall understanability. 
```suggestion
            <>{status === "intent" ? "" : "TX Hash"}</>
```

Reply:
Fixed in 2f7773ea3. Removed the col header title in favor of letting the links speak for themselves.